riverctl/main.zig

```diff
@@ -64,7 +64,15 @@ fn _main() !void {
     // Skip our name, send all other args
     // This next line is needed cause of https://github.com/ziglang/zig/issues/2622
     const args = os.argv;
-    for (args[1..]) |arg| control.addArgument(arg);
+    for (args[1..]) |arg| {
+        if (mem.eql(u8, mem.span(arg), "-version")) {
+            const stdout = std.io.getStdOut().writer();
+            try stdout.print("{s}\n", .{@import("build_options").version});
+            std.os.exit(0);
```

Comment:
I think we should put this check before the for loop and require `-version` to be the first argument. `riverctl send-layout-cmd -version` seems confusing.

Reply:
Yeah makes sense didn't though about this case